app/backend/core/modelhelper.py

```diff
@@ -39,14 +43,19 @@ def num_tokens_from_messages(message: dict[str, str], model: str) -> int:
 
     encoding = tiktoken.encoding_for_model(get_oai_chatmodel_tiktok(model))
     num_tokens = 2  # For "role" and "content" keys
-    for key, value in message.items():
+    for value in message.values():
         if isinstance(value, list):
-            for v in value:
-                # TODO: Update token count for images https://github.com/openai/openai-cookbook/pull/881/files
-                if isinstance(v, str):
-                    num_tokens += len(encoding.encode(v))
-        else:
+            for item in value:
```

Comment:
From https://github.com/openai/openai-cookbook/pull/881/files

Reply:
same - please cite this in a comment